### ISSUE TITLE
[Replacement]: `eslint-plugin-jsx-a11y`

### ISSUE THREAD

**Issue by connorshea** [es-tooling/module-replacements#661]
### Package to replace

eslint-plugin-jsx-a11y

### Suggested replacement(s)

- [eslint-plugin-jsx-a11y-x](https://github.com/es-tooling/eslint-plugin-jsx-a11y-x), has _many_ fewer dependencies and properly, fully supports ESLint v10 / flat config
- [oxlint](https://oxc.rs/docs/guide/usage/linter.html), which is a linter written in Rust and supports [every jsx-a11y rule](https://github.com/oxc-project/oxc/issues/1141) (except the deprecated one). It's much faster and also has fewer dependencies.

Obligatory disclaimer: I'm on the oxlint core team

### Manifest type

preferred (lighter or more modern alternative package)

### Rationale

- The package supports very old versions of ESLint/Node for no good reason and therefore has far, far more dependencies than it needs.
- Both alternative packages have fewer dependencies, and Oxlint is also considerably faster to run, especially on larger projects.

### Code example (optional)

```js
For jsx-a11y-x my understanding is that it should be basically a drop-in replacement if you want to continue using ESLint, just update `jsx-a11y` to `jsx-a11y-x` where necessary: https://github.com/es-tooling/eslint-plugin-jsx-a11y-x#installation

https://github.com/oxc-project/oxlint-migrate for migrating to oxlint
```

**Comment by 43081j** [comment]
im going to mark this as accepted but we should note in the docs that eslint-plugin-jsx-a11y-x is "experimental". it isn't really but there's still work going on to modernise it so i'd rather note that down than not.

cc @jdufresne FYI
